root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -498,6 +498,27 @@ const {LINK_TYPES, cleanURL, guessType, validationRules} = require('../../edit/U
                     expected_clean_url: 'https://www.beatport.com/release/pryda-10-vol-i/1563118',
                only_valid_entity_types: ['release']
         },
+        {
+                             input_url: 'http://classic.beatport.com/release/4/2361374',
+                     input_entity_type: 'release',
+            expected_relationship_type: 'downloadpurchase',
+                    expected_clean_url: 'https://www.beatport.com/release/4/2361374',
+               only_valid_entity_types: ['release']
+        },
+        {
+                             input_url: 'https://www.beatport.com/en-US/html/content/release/detail/161035/Back%20To%20The%20Future',
+                     input_entity_type: 'release',
+            expected_relationship_type: 'downloadpurchase',
+                    expected_clean_url: 'https://www.beatport.com/release/back-to-the-future/161035',
+               only_valid_entity_types: ['release']
+        },
+        {
+                             input_url: 'https://www.beatport.com/en-US/html/content/release/detail/06130/22%%21%20&tracks#',
```

Comment:
This URL is a 400 error (rather than just a 404, assuming it's made up) - is the URL malformed in some way?

Reply:
Good catch, the error comes from the first `%` sign which is not encoded properly.